uPortal-webapp/src/main/webapp/media/skins/respondr/common/common.less

```diff
@@ -17,17 +17,17 @@
  * under the License.
  */
  /**
-  * NOTE: this file is located outside the less/ directory due to a limitation 
+  * NOTE: this file is located outside the less/ directory due to a limitation
   * in the LESS CSS Maven Plugin's inability to traverse relative paths.
   *
   * Also, due to a limitation in the LESS CSS Maven Plugin, whitespace before an
   * @import will cause a compile error.
   */
- 
+
  /* ==========================================================================*
     Bootstrap
     ==========================================================================*/
-@import "bootstrap/bootstrap.less";
+@import "../../../../../../../node_modules/bootstrap/less/bootstrap.less";
 
```

Comment:
@drewwills Correct me if I have this wrong. Dynamic respondr skins allows a selection of a skin available for the theme. The best practice to create these skins is to derived from defaultSkin, copying the directory and defaultSkin.less file. This file is where common/common.less is imported, which specifies Bootstrap's location.

It seems straightforward that derived skins following this practice would work.

I will be testing this shortly for some client work (but the merge should have waited.)